docs/development_guide/README.md

```diff
@@ -139,7 +138,9 @@ export CTR_REGISTRY_PASSWORD=<your password>
 
 ```
 
-(NOTE: these requirements are true for automatic demo deployment using the available demo scripts; [#1416](https://github.com/openservicemesh/osm/issues/1416) tracks an improvement to not strictly require these and use upstream images from official dockerhub registry if a user does not want/need changes on the code)
+> Note: These requirements are true for automatic demo deployment using the available demo scripts; [#1416](https://github.com/openservicemesh/osm/issues/1416) tracks an improvement to not strictly require these and use upstream images from official dockerhub registry if a user does not want/need changes on the code
+
+> Note: For local development, you need to set the `CTR_TAG` environment variable. Without `CTR_TAG`, commands like `make docker-build` and `make build-osm-all` will fail. Example: `export CTR_TAG=latest-main` . 
```

Comment:
this is surprising as well.  The tag is defaulted in the make file: https://github.com/openservicemesh/osm/blob/444ccbfaf54238d77bb950a32f43233535d4fa31/Makefile#L7

did you modify any of the `env` files?

Reply:
I didn't change the env file. I tested again with main branch again and you are **CORRECT**. I think the error might be resolved by another fix I did: log into docker in command line and log into DockerHub on my Docker desktop. I will change Note. The fix I did was from a [docker issue](https://github.com/docker/build-push-action/issues/160#issuecomment-1121529469).